src/fadec/build.sh

```diff
@@ -6,15 +6,19 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 OUTPUT="${DIR}/../../flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/panel/fadec.wasm"
 
 if [ "$1" == "--debug" ]; then
+  WASMLD_ARGS=""
   CLANG_ARGS="-g"
 else
-  WASMLD_ARGS="--strip-debug"
+  WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
+  CLANG_ARGS="-flto -O2 -DNDEBUG"
 fi
```

Comment:
```suggestion
if [ "$1" == "--debug" ]; then
CLANG_ARGS="-g -DDEBUG"
WASMLD_ARGS="-O0"
else
CLANG_ARGS="-flto -O2"
WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
fi
```
I think it would be more useful to have DEBUG when in debug and nothing when not - wdyt?

Reply:
NDEBUG comes from the MSFS SDK template… I assume it’s used in their headers.

Reply:
Ok - I have added DEBUG on mine as well - so I have both now. Maybe you can do the same or it will be in my PR then.

Reply:
I'd rather keep this PR applicable to what's in master now, then do stuff for your PR in that branch.